mockmpi/comm.py

```diff
@@ -113,7 +113,7 @@ def allreduce(self, sendobj, op=None):
         return d
 
     def Reduce(self, sendbuf, recvbuf, op=None, root=0):
-        if sendbuf is IN_PLACE:
+        if (sendbuf is IN_PLACE) or (isinstance(sendbuf, int) and sendbuf == IN_PLACE):
```

Comment:
Can it just be `sendbuf == IN_PLACE`.  Seems simpler and probably equivialent?

Reply:
Annoyingly no, as then you get `The truth value of an array with more than one element is ambiguous` when sendbuf is a numpy array.

Reply:
Could remove the first half of the test of course, I'll do that.